snapcraft/plugins/_python/_python_finder.py

```diff
@@ -0,0 +1,67 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import glob
+
+from . import errors
+
+
+def get_python_command(python_version, stage_dir, install_dir):
```

Comment:
how about

```python
def get_python_command(python_major_version, *, stage_dir, install_dir)
```

Where `python_major_version` can be 2 or 3? You can expand the expected command from a dict if needed.